app/products/calls/components/call_message/call_message.tsx

```diff
@@ -117,37 +116,26 @@ const CallMessage = ({post, user, teammateNameDisplay, confirmToJoin, alreadyInT
                     style={style.phoneHangupIcon}
                 />
                 <View style={style.messageText}>
-                    <FormattedText
-                        id='call_message.call_ended'
-                        defaultMessage='Call ended'
+                    <Text
                         style={style.startedText}
-                    />
+                    >{'Call ended'}</Text>
```

Comment:
Why is this removing the translations?

Reply:
Cause due to this code being embedded in the actual product, community members started translating things which were far from well defined so we decided to defer translation support to a later time mostly to avoid having community spend time on strings that may change at any time. More context can be found in https://community-daily.mattermost.com/core/pl/qoztij6tr7nwxj8gntzxajc8ja

Reply:
Fair enough.